go-controller/pkg/metrics/ovnkube_controller.go

```diff
@@ -269,6 +269,23 @@ var metricEgressFirewallCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The number of egress firewall policies",
 })
 
+/** AdminNetworkPolicyMetrics Begin**/
```

Comment:
whats your goal with these metrics? what do you hope to achieve and how do you want the user to use them?

Reply:
Great question which I should have put in the PR description,
My goal with these two metrics are:

1. understand how the user is using this feature
2. how many admin network policies have they created (I plan to do future PRs for more fine-grained information)
3. is the user using baseline admin network policy in their cluster?

So its also for the user but more so for us.

Reply:
Do you plan to create any alerts based on these metrics? For example, if theres a limit for any of the CRs, you can put a warning when they are closed to the threshold.

Reply:
I think I remember reading that theres a limit on at least one of the CRs.

Reply:
yeah there is a limit on ANP being max at 100, but TBH, I cannot fathom anyone using more than 10 :D so perhaps alerts is definitely good advice in case someone abuses this (as is always case with all features!)

Reply:
API enforces this anyway right - so alert might not be needed.

Reply:
ah no API doesn't enforce it (API says max 1000), its an ovnkube thing where I restrict them being able to do only 100 priorities so 100 ANPs, alert might be needed here actually.